.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Ensure issue is authorized
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  ensure-issue-authorization:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: exercism/kntsoriano-test@v1
```

Comment:
@iHiD This points to the repository you've created. Can we rename it into `exercism/ensure-issue-authorization`?